.github/workflows/manual-nixos.yml

```diff
@@ -1,9 +1,8 @@
 name: "Build NixOS manual"
 
-permissions: read-all
-
```

Comment:
Let's keep these: these jobs have no need for more than read.

Reply:
Alright!